composer.json

```diff
@@ -7,9 +7,9 @@
         "webexcess/redirecthandler-backend": "*"
     },
     "require": {
-        "php": ">=7.4",
-        "neos/redirecthandler": "~5.0 || dev-main",
-        "neos/neos": "~7.3 || ~8.0 || dev-master",
+        "php": ">= 8.2",
```

Comment:
I think you based your change on the incorrect branch. Please make sure your commit is on top of the 2.x branch and not on the neos 9 branch.

Reply:
I'll close this pull request and submit a new one with the proper base branch.